report-job-status/action.yml

```diff
@@ -20,6 +20,13 @@ inputs:
       It must be always `ToJson(steps)` expression if you provide steps info
     required: false
     default: '{}'
+  job-name:
+    description: >
+      Full job name with matrix combinations. For example, job `test` with
+      matrix options `foo='a'` and `bar='b'` will be named `test (a, b)`.
+      This is required for finding exact job id of matrix jobs.
```

Comment:
OK, we need it to get IDs, but for what IDs are used?

Reply:
added a line about ID usage